api/tests/acceptance/application/feedbacks/feedback-controller_test.js

```diff
@@ -40,7 +40,7 @@ describe('Acceptance | Controller | feedback-controller', function () {
             },
           },
         },
-        headers: { 'user-agent': 'Firefox rocks', authorization: generateValidRequestAuthorizationHeader() },
```

Comment:
Ce header est effectivement inutile, mais on test spécifiquement que ca marche sans un peu plus loin.

Reply:
Le découpage des commit n'est pas top. Souhaites-tu que je le corrige ?

Reply:
Je parle du test "should return 201 HTTP status code when missing authorization header" un peu plus bas dans le fichier.

Reply:
Corrigé